src/documents/consumer.py

```diff
@@ -790,6 +797,7 @@ def _store(
         self,
         text: str,
         date: Optional[datetime.datetime],
+        pages_count: int,
```

Comment:
This could be an `None` since not all parsers implement a page count

Reply:
I think he's saying this should be `Optional[int]`

Reply:
[Update pages_count to Optional[int] in consumer](https://github.com/paperless-ngx/paperless-ngx/pull/7750/commits/acec8e35fa450aab864556d948346453d091fc0b)